etcdctl/ctlv3/command/printer.go

```diff
@@ -131,13 +131,17 @@ func (s *simplePrinter) MemberList(resp v3.MemberListResponse) {
 
 func (s *simplePrinter) MemberStatus(statusList []statusInfo) {
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"endpoint", "ID", "version"})
+	table.SetHeader([]string{"endpoint", "ID", "version", "store size", "raft index", "raft term", "raft lead"})
```

Comment:
probably just db size? then leader, raft term, raft index? also we should use something like https://gowalker.org/github.com/dustin/go-humanize to format db size in bytes.